### DISCUSSION TITLE
RSS o nowych ofertach

### DISCUSSION THREAD

**Discussion by adamz464** [allegro/allegro-api#4635] [Suggestions - rejected]
Uważam, że ten wątek nie ma szans na powodzenie: https://github.com/allegro/allegro-api/discussions/4584

Dlatego lepiej aby udostępniono RSS na aktywnym wyszukiwaniu. Udostępniane informacje są minimalne, ale wystarczajace do tych potrzeb.

Nawet pazerny Ebay wie o tym, że taka funkcjonalność przyczynia się do wzrostu sprzedaży (czy to kup-teraz schodzące od ręki czy to licytacje, które pójdą wyżej ze względu na większe grono zainteresowanych).
Jest to wystarczający i solidny argument.

**Comment by maciekjaros** [comment]
Popieram. Jakiekolwiek źródło danych, które odświeża się dostatecznie często i można odpytywać je z podobną częstotliwością jak API REST bez blokady IP.

**Comment by juhacz** [comment]
Oni nie mają w zwyczaju przywracania tego co kiedyś zabrali ... RSS kiedyś było, ale już nie ma. Jedynie co zabrali i po wielkiej burzy przywrócili to wyszukiwanie w zakończonych.

**Comment by hahyes** [comment]
> Nawet pazerny Ebay wie o tym

Ebay technologicznie żyje w poprzedniej dekadzie, marny przykład. Zwłaszcza, że ten sam Ebay nie potrafił zrobić czegokolwiek na polskim rynku. RSS to jest jednak przeżytek, tak jak SOAP.

A o wyszukiwarce zapomnijcie, Allegro dostosowuje REST API do swojej logiki biznesowej, a nie do waszej. W ich interesie jest, żebyście wykupili ich usługi. Tylko tyle i aż tyle. 

Z RSSem (jak i czymkolwiek innym, co miałoby zastępować wyszukiwarkę) jest jeden problem - ktoś to musi utrzymywać. To musi zostać napisane, ktoś musi programistom zapłacić. Czy pieniądze wyłożone w wasze zadowolenie zwiększą wpływy dla Allegro? 99%, że nie. Tak działa korpo, tutaj nie ma miejsca na poświęcanie się w imię idei - o ile idea nie daje profitów (jak kolorowanie logotypów w 6 kolorów). A w tym wypadku nie da im żadnych, a wręcz wywołuje konflikt interesów. 

Tak jak sercem jestem za tym pomysłem i każdemu podbiłem post, tak zdroworozsądkowo to ma małe szanse, że znajdzie odzew gdzieś w strukturach Allegro. Wierzę, że deweloperzy od REST API te dyskusje przekazują dalej. Tylko na szczeblu decyzji biznesowych to jest zapewne pyłek w oku. Smutna rzeczywistość.

**Reply by adamz464** [reply]
bełkot.
Udostępnienie RSS czy to JSON (np marktplaats) przyczynią się oczywiście do wpływów allegro, poprzez zwiększenie zainteresowania aukcjami a w efekcie wyższymi prowizjami.
API może być też płatne, ale nie jest.

Nakład sił na takiego RSSa (nawet w jakimś standardzie korpo) to bez przesady (porównując ich pomysły na siłę wprowadzane)

**Reply by hahyes** [reply]
> Udostępnienie RSS czy to JSON (np marktplaats) przyczynią się oczywiście do wpływów allegro, poprzez zwiększenie zainteresowania aukcjami a w efekcie wyższymi prowizjami.

Jak się niby przyczynią? Jak RSS zwiększa zainteresowanie aukcjami? W jaki sposób RSS spowoduje większe prowizje? Wyjaśnij zamiast rzucać frazesami ułożonymi w bełkot. Nie kupuje bajek o tym, że sprzedawcy magicznie wykorzystają moc takiego katalogu i zapewnią Allegro cokolwiek. Zwłaszcza, że mówimy o małych sprzedawcach, gdzie Allegro pokazało dokładnie, gdzie ich ma oraz wpływy od nich. Mniej bajek, więcej konkretów. 

> Nakład sił na takiego RSSa (nawet w jakimś standardzie korpo) to bez przesady (porównując ich pomysły na siłę wprowadzane)

Jeżeli ktoś pracuje nad RSSem, nie pracuje nad innymi i ważniejszymi z punktu widzenia modelu biznesowego projektami. Dla wyższego szczebla to będzie wystarczający powód, żeby takie pomysły mieć zwyczajnie gdzieś.

**Reply by adamz464** [reply]
to, że nie magluję cały dzień co kilka minut ręcznie wyszukiwarką kilkaset fraz. Omijają mnie aukcje = nie licytuję i przedmiot schodzi taniej. A nie jestem jedyny a wszyscy co oczekują powrotu api (oraz potencjalni ich klienci jak ktoś robił z tego komercję w co nie wątpię)

**Comment by MartaNowaczyk** [comment]
Zaktualizowaliśmy status sugestii jako "odrzucona", ponieważ nie planujemy takiego rozwiązania.
